src/components/Trends/Explore/TrendsExploreAdditionalInfo.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react'
+import { compose } from 'recompose'
+import { graphql } from 'react-apollo'
+import { Tabs } from '@santiment-network/ui'
+import { DAY, getTimeIntervalFromToday } from '../../../utils/dates'
+import { ALL_INSIGHTS_BY_TAG_QUERY } from '../../../queries/InsightsGQL'
+import { NEWS_QUERY } from '../../News/NewsGQL'
+import {
+  getInsightTrendTagByDate,
+  oneDayTimestamp
+} from '../../Insight/InsightsTrends'
+import InsightCard from '../../Insight/InsightCard'
+import News from '../../News/News'
+import styles from './TrendsExploreAdditionalInfo.module.scss'
+
+const NEWS_INDEX = 'News'
+const INSIGHTS_INDEX = 'Insights'
+
+const TrendsExploreAdditionalInfo = ({ news, insights }) => {
+  let [selectedTab, setSelectedTab] = useState(null)
+
+  function handleSelectTab (tab) {
+    if (tab !== (selectedTab || defaultSelectedTab)) {
+      setSelectedTab(tab)
+    }
+  }
+
+  const newsTab = {
+    index: NEWS_INDEX,
+    content: `${NEWS_INDEX} (${news.length})`
+  }
+
+  const insightsTab = {
+    index: INSIGHTS_INDEX,
+    content: `${INSIGHTS_INDEX} (${insights.length})`
+  }
+
+  const tabs = []
+  if (insights.length > 0) tabs.push(insightsTab)
+  if (news.length > 0) tabs.push(newsTab)
+
+  let defaultSelectedTab
+
+  if (tabs.length === 0) return null
+  else defaultSelectedTab = tabs[0].index
+
+  return (
+    <section className={styles.wrapper}>
+      <Tabs
+        options={tabs}
+        defaultSelectedIndex={defaultSelectedTab}
+        onSelect={handleSelectTab}
+      />
+      <div className={styles.tabsContent}>
+        {(selectedTab || defaultSelectedTab) === NEWS_INDEX && (
+          <News data={news} />
+        )}
+        {(selectedTab || defaultSelectedTab) === INSIGHTS_INDEX && (
+          <div className={styles.insights}>
+            {insights.map(insight => (
+              <InsightCard
+                small
+                grey
+                withAuthorPic={false}
+                {...insight}
+                key={insight.id}
+                className={styles.insight}
+              />
+            ))}
+          </div>
+        )}
+      </div>
+    </section>
+  )
+}
+
+const getTrendsTags = numberOfLastDays => {
```

Comment:
@haritonasty You can write it without an imperative style of code.
return Array.from({length: numberOfLAstDays}, (_, index) => additionalLogic(index))

Reply:
@haritonasty We have this function somewhere, just import it

Reply:
@DmitriVanGuard didn't find mentions `Array.from`... Create it in `utils.js`?

Reply:
Export and use this function

https://github.com/santiment/app/blob/93632350aab667c249dfcc3b290f88a3f2d570f4/src/components/Insight/InsightsTrends.js#L22